src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -75,15 +71,8 @@ internal sealed class AnalysisConfigMonitor : IAnalysisConfigMonitor, IDisposabl
 
         private void OnUserSettingsChanged(object sender, EventArgs e)
         {
-            if (activeSolutionBoundTracker.CurrentConfiguration.Mode == SonarLintMode.Standalone)
-            {
-                logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
-                OnSettingsChangedAsync().Forget();
-            }
-            else
-            {
-                logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsIgnoredForConnectedModeLanguages);
-            }
+            logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
```

Comment:
I think its fine to leave this message as if user specified settings are not supported it'll print this later "[RuleSettings] Using connected mode settings. User-specified settings in settings.json will be ignored."